### ISSUE TITLE
Disable converter in inline toolbar for some specific tools

### ISSUE THREAD

**Issue by Malimalo** [codex-team/editor.js#2781]
I developed a simple tool to have some thing structured and formatted with multiple simple text fields.
I don't want to handle the conversion of this type of block and I want to keep the inline toolbar for formatting.

I tried multiple things and I ended up adding custom styles to hide the converter tool button:
```
/* Hide the "Convert to" button */
button[data-item-name="convert-to"] {
  display: none !important;
}

/* Hide the separator */
.ce-popover-item-separator {
  display: none !important;
}
```

I've seen #2262 and I want to de exactly the opposite. In the documentation, the converter is not shown on the screenshots...
![image](https://github.com/user-attachments/assets/5d166c89-9d26-478c-b1c7-78a3fce23c6b)

How can I do this ?

**Comment by minhduc68** [comment]
+1 Same issue here. For me, the converter is not useful with the image block since the end user can simply delete it instead of converting it to another block type. Therefore, I want to hide the converter (see the image below)
<img width="758" alt="Screenshot 2024-07-18 at 10 50 33" src="https://github.com/user-attachments/assets/7cb04cdd-0088-452f-90aa-0a9c8726d25f">

**Comment by neSpecc** [comment]
Resolved by #2799
